Sources/SwiftUITooltip/TooltipViewExtension.swift

```diff
@@ -8,27 +8,27 @@
 import SwiftUI
 
 public extension View {
-    func tooltip<TooltipContent: View>(@ViewBuilder content: @escaping () -> TooltipContent) -> some View {
+    func tooltip<TooltipContent: View>(_ enabled: Binding<Bool> = .constant(true), @ViewBuilder content: @escaping () -> TooltipContent) -> some View {
         let config: TooltipConfig = DefaultTooltipConfig.shared
 
-        return modifier(TooltipModifier(config: config, content: content))
+        return modifier(TooltipModifier(enabled: enabled, config: config, content: content))
     }
 
-    func tooltip<TooltipContent: View>(config: TooltipConfig, @ViewBuilder content: @escaping () -> TooltipContent) -> some View {
-        modifier(TooltipModifier(config: config, content: content))
+    func tooltip<TooltipContent: View>(_ enabled: Binding<Bool> = .constant(true), config: TooltipConfig, @ViewBuilder content: @escaping () -> TooltipContent) -> some View {
+        modifier(TooltipModifier(enabled: enabled, config: config, content: content))
     }
 
-    func tooltip<TooltipContent: View>(_ side: TooltipSide, @ViewBuilder content: @escaping () -> TooltipContent) -> some View {
+    func tooltip<TooltipContent: View>(_ enabled: Binding<Bool> = .constant(true), side: TooltipSide, @ViewBuilder content: @escaping () -> TooltipContent) -> some View {
         var config = DefaultTooltipConfig.shared
         config.side = side
 
-        return modifier(TooltipModifier(config: config, content: content))
+        return modifier(TooltipModifier(enabled: enabled, config: config, content: content))
     }
     
-    func tooltip<TooltipContent: View>(_ side: TooltipSide, config: TooltipConfig, @ViewBuilder content: @escaping () -> TooltipContent) -> some View {
+    func tooltip<TooltipContent: View>(_ enabled: Binding<Bool> = .constant(true), side: TooltipSide, config: TooltipConfig, @ViewBuilder content: @escaping () -> TooltipContent) -> some View {
         var config = config
         config.side = side
 
-        return modifier(TooltipModifier(config: config, content: content))
+        return modifier(TooltipModifier(enabled: enabled, config: config, content: content))
```

Comment:
I would add these 4 `tooltip` functions in addition to the ones that existed before and would just add `.constant(true)` in the old ones as the first argument.